functionalDB/chapter.md

```diff
@@ -871,7 +876,7 @@ Locating the index of the joining variable is done by *index-of-joining-variable
          collapsed (reduce collapsing-fn metas-seq)] 
      (first (keep-indexed #(when (variable? %2 false) %1)  collapsed)))) 
 ````
-We begin by extracting the metadata of each clause in the query. This metadata is a 3-element vector, each of which is a variable name or *nil*, and reduces them to produce a single variable name or *nil*. If a variable name is produced, this means it appears in all of the metadata vectors at the same index -- i.e. this is the joining variable. We can thus choose the appropriate index by using the mapping described above. 
+We begin by extracting the metadata of each clause in the query. This metadata is a 3-element vector, each element of which is a variable name or `nil`, and reduces them to produce a single variable name or `nil`. If a variable name is produced, this means it appears in all of the metadata vectors at the same index; i.e., this is the joining variable. We can thus choose the appropriate index by using the mapping described above. 

```

Comment:
Right, here is a (hopefully) better version:
We begin by extracting the metadata of each clause in the query. This extracted metadata is a 3-element vector, each element in that vector is either a variable name or nil (note that there is no more then one variable name in that vector). Once the vector is extracted, we produce from it (by reducing it) a single value, which is either a variable name or nil.